tutorials/ComplexArithmetic/Workbook_ComplexArithmetic.ipynb

```diff
@@ -0,0 +1,953 @@
+   "source": [
+    "# Complex Arithmetic Kata Workbook\n",
+    "\n",
+    "**What is this workbook?**\n",
+    "A workbook is a collection of problems, accompanied by solutions to them. \n",
+    "The explanations focus on the logical steps required to solve a problem; they illustrate the concepts that need to be applied to come up with a solution to the problem, explaining the mathematical steps required. \n",
+    "\n",
+    "Note that a workbook should not be the primary source of knowledge on the subject matter; it assumes that you've already read a tutorial or a textbook and that you are now seeking to improve your problem-solving skills. You should attempt solving the tasks of the respective kata first, and turn to the workbook only if stuck. While a textbook emphasizes knowledge acquisition, a workbook emphasizes skill acquisition.\n",
+    "\n",
+    "This workbook describes the solutions to the problems offered in the [Complex Arithmetic kata](./ComplexArithmetic.ipynb). \n",
+    "Since the tasks are offered as programming problems, the explanations also cover some elements of Python that might be non-obvious for a first-time user.\n",
+    "\n",
+    "**What you should know for this workbook**\n",
+    "\n",
+    "1. Basic math\n",
+    "2. Basic Python knowledge is usefull but not needed\n",
+    "\n",
+    "If you need to look up some formulas quickly, you can find them in [this cheatsheet](https://github.com/microsoft/QuantumKatas/blob/master/quickref/qsharp-quick-reference.pdf).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook has several tasks that require you to write Python code to test your understanding of the concepts. If you are not familiar with Python, [here](https://docs.python.org/3/tutorial/index.html) is a good introductory tutorial for it. \n",
+    "\n",
+    "Let's start by importing some useful mathematical functions and constants, and setting up a few things necessary for testing the exercises. **Do not skip this step**.\n",
+    "\n",
+    "Click the cell with code below this block of text and press `Ctrl+Enter` (`⌘+Enter` on Mac)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Success!\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Run this cell using Ctrl+Enter (⌘+Enter on Mac).\n",
+    "from testing import exercise\n",
+    "from typing import Tuple\n",
+    "\n",
+    "import math\n",
+    "\n",
+    "Complex = Tuple[float, float]\n",
+    "Polar = Tuple[float, float]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Algebraic Perspective\n",
+    "\n",
+    "## Imaginary numbers\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 1</span>: Powers of $i$.\n",
```

Comment:
Could you please add links from the tutorial exercises to the corresponding solutions here? Currently the reader can use links in the workbook to return to the tutorial but not vice versa.